src/client/extension.ts

```diff
@@ -34,7 +33,7 @@ export async function activate(
     _context = context;
 
     // setup telemetry
-    _telemetry = new TelemetryReporter(context.extension.id, context.extension.packageJSON.version, AI_KEY);
+    _telemetry = new TelemetryReporter(context.extension.id, context.extension.packageJSON.version, context.extension.packageJSON.aiKey);
```

Comment:
please undo this change. It diverges from how we expect to model selecting the AI resource that belongs to the region configured by the user.